packages/psnotify.vm/tools/chocolateyinstall.ps1

```diff
@@ -10,38 +10,31 @@ try {
 
     $unzipLocation = 'C:\'  # psnotify has a requirement of being located in 'C:\psnotify'
```

Comment:
why can't you use the helper function with this unzip location instead of duplicating the code? what else are you changing to need this duplication?

Reply:
I needed to add `-WorkingDirectory $executableDir` to creating the shortcut, which was done in the other PR. 

It didn't seem like a change that needed to be made for every package going forward at the moment, but if it is, we can make another PR for that as well.